frontend/packages/noobaa-storage-plugin/src/components/activity-card/activity-card.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import { EventKind } from '@console/internal/module/k8s';
+import { FirehoseResource, FirehoseResult } from '@console/internal/components/utils';
+import { EventModel } from '@console/internal/models';
+import ActivityBody, {
+  RecentEventsBody,
+  OngoingActivityBody,
+} from '@console/shared/src/components/dashboard/activity-card/ActivityBody';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { DATA_RESILIENCE_QUERIES } from '../../queries';
+import { NooBaaObjectBucketClaimModel, NooBaaObjectBucketModel } from '../../models';
+import { isDataResiliencyActivity } from './data-resiliency-activity';
+
+const eventsResource: FirehoseResource = { isList: true, kind: EventModel.kind, prop: 'events' };
+
+const ocsEventNamespaceKindFilter = (event: EventKind): boolean =>
+  _.get(event, 'involvedObject.kind') ===
+  (NooBaaObjectBucketClaimModel.kind || NooBaaObjectBucketModel.kind);
```

Comment:
This cannot be only OB and OBC's we need to register bucket class and backing store events too. There are models for them already. Please add them.

Reply:
The events are confirmed to come for OB/OBC for now FYI